tests/integration/components/frost-modal-about-dialog-test.js

```diff
@@ -8,43 +11,219 @@ import {
   describeComponent,
   it
 } from 'ember-mocha'
-import hbs from 'htmlbars-inline-precompile'
-import { beforeEach } from 'mocha'
+import {
+  initialize as initializeSvgUse
+} from 'ember-frost-core/instance-initializers/svg-use-polyfill'
 
+import hbs from 'htmlbars-inline-precompile'
+import {
+  beforeEach
+} from 'mocha'
+import Ember from 'ember'
 describeComponent(
   'frost-modal-about-dialog',
-  'Integration: FrostModalAboutDialogComponent',
-  {
+  'Integration: FrostModalAboutDialogComponent', {
     integration: true
   },
   function () {
     beforeEach(function () {
       initialize()
+      initializeSvgUse()
     })
 
     // TODO Get ember-cli-visual-acceptance plugged in before PR
-    it('renders', function () {
+    it('renders', function (done) {
+      this.timeout(10000)
+      this.set('closeModal', () => {
+        this.set('isModalVisible', false)
+      })
+      this.set('isModalVisible', true)
+
+      this.render(hbs `
+        {{frost-modal-outlet}}
+
+        {{frost-modal-about
+          brandingStrip=(hash
+            icon='branding-strip'
+            pack='dummy'
+          )
+          copyright='copyright'
+          hook='about-dialog'
+          isVisible=isModalVisible
+          logo=(hash
+            icon='logo'
+            pack='dummy'
+          )
+          product=(hash
+            icon='product'
+            pack='dummy'
+          )
+          versions=(array
+            'Version: 1.0.0'
+          )
+          onClose=(action closeModal)
+        }}
+      `)
+        // eslint-disable-next-line
+      expect($hook('about-dialog-modal'), 'Is modal visible')
+        .to.have.length(1)
+      Ember.run.later(function () {
+        return capture('about', {
+          targetElement: this.$('.frost-modal-outlet-container.about')[0],
```

Comment:
I could but then I wouldn't capture the container and the svg.

<img width="188" alt="screen shot 2016-08-26 at 2 32 38 pm" src="https://cloud.githubusercontent.com/assets/9057680/18016381/14b8985c-6b9a-11e6-8b4a-b9270d79bd63.png">

Current uses `targetElement: $hook('about-dialog-modal'),`